LogsService.RabbitMQ/Context/IRabbitMqContext.cs

```diff
@@ -0,0 +1,10 @@
+﻿using RabbitMQ.Client;
+
+namespace LogsService.RabbitMQ.Context
+{
+    public interface IRabbitMqContext
+    {
+        IConnection Connection { get; }
+        IConnection GetRabbitConnection();
```

Comment:
It seems strange that you have property and function, which configure that property at the same time in the interface.

Reply:
because this interface can get property IConnection and return new IConnection from the method